crates/rooch-types/src/chain_id.rs

```diff
@@ -178,6 +198,11 @@ impl BuiltinChainID {
     pub fn genesis_ctx(&self) -> GenesisContext {
         let chain_id = self.chain_id().id;
         match self {
+            BuiltinChainID::Local => {
+                //Local timestamp from 0, developer can manually set the timestamp
+                let timestamp = 0;
+                GenesisContext::new(chain_id, timestamp)
+            }
             BuiltinChainID::Dev => {
                 //Dev timestamp from 0, developer can manually set the timestamp
```

Comment:
This should be dev chain's timestamp.

Reply:
Set as the test network, use a Ethereum test net block height timestamp.